tools-v2/pkg/cli/command/curvebs/update/leader_schedule/leader_schedule.go

```diff
@@ -0,0 +1,165 @@
+	return uRpc.Client.RapidLeaderSchedule(ctx, uRpc.Request)
+}
+
+type LeaderScheduleCommand struct {
+	basecmd.FinalCurveCmd
+
+	Rpc *UpdateLeaderScheduleRpc
+}
+
+var _ basecmd.FinalCurveCmdFunc = (*LeaderScheduleCommand)(nil) // check interface
+
+const (
+	leaderScheduleExample = `$ curve bs update leader-schedule --logicalpoolid 1`
+)
+
+func NewLeaderScheduleCommand() *cobra.Command {
+	return NewUpdateLeaderScheduleCommand().Cmd
+}
+
+func (lCmd *LeaderScheduleCommand) AddFlags() {
+	config.AddBsMdsFlagOption(lCmd.Cmd)
+	config.AddRpcRetryTimesFlag(lCmd.Cmd)
+	config.AddRpcTimeoutFlag(lCmd.Cmd)
+
+	config.AddBsLogicalPoolIdRequiredFlag(lCmd.Cmd)
+}
+
+func NewUpdateLeaderScheduleCommand() *LeaderScheduleCommand {
+	sCmd := &LeaderScheduleCommand{
+		FinalCurveCmd: basecmd.FinalCurveCmd{
+			Use:     "leader-schedule",
+			Short:   "rapidly transfer leader",
+			Example: leaderScheduleExample,
+		},
+	}
+
+	basecmd.NewFinalCurveCli(&sCmd.FinalCurveCmd, sCmd)
+	return sCmd
+}
+
+func (lCmd *LeaderScheduleCommand) Init(cmd *cobra.Command, args []string) error {
+	mdsAddrs, err := config.GetBsMdsAddrSlice(lCmd.Cmd)
+	if err.TypeCode() != cmderror.CODE_SUCCESS {
+		return err.ToError()
+	}
+
+	header := []string{cobrautil.ROW_RESULT, cobrautil.ROW_REASON}
+	lCmd.SetHeader(header)
+
+	timeout := config.GetFlagDuration(lCmd.Cmd, config.RPCTIMEOUT)
+	retryTimes := config.GetFlagInt32(lCmd.Cmd, config.RPCRETRYTIMES)
+	logicalPoolID := config.GetBsFlagUint32(lCmd.Cmd, config.CURVEBS_LOGIC_POOL_ID)
+
+	lCmd.Rpc = &UpdateLeaderScheduleRpc{
+		Request: &schedule.RapidLeaderScheduleRequst{
+			LogicalPoolID: &logicalPoolID,
+		},
+		Info: basecmd.NewRpc(mdsAddrs, timeout, retryTimes, "RapidLeaderSchedule"),
+	}
+
+	return nil
+}
+
+func (lCmd *LeaderScheduleCommand) Print(cmd *cobra.Command, args []string) error {
+	return output.FinalCmdOutput(&lCmd.FinalCurveCmd, lCmd)
+}
+
+func (lCmd *LeaderScheduleCommand) RunCommand(cmd *cobra.Command, args []string) error {
+	result, err := basecmd.GetRpcResponse(lCmd.Rpc.Info, lCmd.Rpc)
+	if err.TypeCode() != cmderror.CODE_SUCCESS {
+		return err.ToError()
+	}
+	out := make(map[string]string)
+	if response, ok := result.(*schedule.RapidLeaderScheduleResponse); ok {
+		if response.GetStatusCode() != 0 {
+			out[cobrautil.ROW_RESULT] = cobrautil.ROW_VALUE_FAILED
+			out[cobrautil.ROW_REASON] = statuscode.SchStatusCode_name[*response.StatusCode]
+		} else {
+			out[cobrautil.ROW_RESULT] = cobrautil.ROW_VALUE_SUCCESS
+			out[cobrautil.ROW_REASON] = cobrautil.ROW_VALUE_NULL
+		}
+	}
+
+	res := cobrautil.Map2List(out, lCmd.Header)
+	lCmd.TableNew.Append(res)
+
+	lCmd.Result = out
+	lCmd.Error = cmderror.ErrSuccess()
+	return nil
+}
+
+func (lCmd *LeaderScheduleCommand) ResultPlainOutput() error {
+	return output.FinalCmdOutputPlain(&lCmd.FinalCurveCmd)
+}
+
+func UpdateLeaderSchedule(caller *cobra.Command) (*tablewriter.Table, *cmderror.CmdError) {
```

Comment:
return a `*interface{}` is better(rather than tablewriter.Table).
just like fellow:
https://github.com/opencurve/curve/blob/e770d4aae4395237efdee829235ee3569bef4a2d/tools-v2/pkg/cli/command/curvebs/status/mds/mds.go#L198-L212